app/routes.test.js

```diff
@@ -92,6 +93,7 @@ describe('routes', () => {
 
       return request(setUpFakeApp())
         .get('/')
+        .send({ _csrf: 'dada' })
```

Comment:
should we make the _csrf less random? haha

Reply:
ooops that's worrying. That shouldn't be there at all.

Reply:
good spot 👍